### PR TITLE
Launch Image Carousel

### PR DESCRIPTION
**Pull request by Celinarabe**
I used the `PageView` element to contain the image carousel. I created the `LaunchCarouselAdapter` which extends PageAdapter to create the slideshow effect. If there are no photos available, the default SpaceX logo ImageView will be visible.

https://user-images.githubusercontent.com/41392379/166297659-2a9d9e06-2130-4e64-adea-f570ce6cccb5.mov

### CODE REVIEW

**Comment by joelc1225** [comment]
Beautiful and smooth! Love it. LGTM. Waiting on second approval before merging.

File: app/src/main/java/com/example/voyagerx/adapters/LaunchCarouselAdapter.kt
Comment (brownyte) [inline app/src/main/java/com/example/voyagerx/adapters/LaunchCarouselAdapter.kt:19]: @Celina, what do the ` around object represent (`object`)?

File: app/src/main/res/layout/fragment_launch_details.xml
Comment (brownyte) [inline app/src/main/res/layout/fragment_launch_details.xml:32]: @Celina, further down on this xml file, your imageButtons have still have their default names. All of your code is so meticulously done, I really have no comments lol

**Review by brownyte — COMMENTED** [review]
@Celinarabe, I am curious how `object` works, so I left a comment where it was used. Looks beautiful otherwise!

**Review by VivianLi98 — APPROVED** [review]
YAY great work!

**Comment by chrsbell** [comment]
This looks great!

File: app/src/main/res/layout/fragment_launch_details.xml
Comment (Celinarabe) [reply app/src/main/res/layout/fragment_launch_details.xml:32]: Thanks for the catch! Updated here: [a48d194](https://github.com/onramp-io/twitch-android-project-2/pull/20/commits/a48d194ce83bc6f7975a2b2e23beecaa5208f9a0)

File: app/src/main/java/com/example/voyagerx/adapters/LaunchCarouselAdapter.kt
Comment (Celinarabe) [reply app/src/main/java/com/example/voyagerx/adapters/LaunchCarouselAdapter.kt]: The backticks `` can be used to escape Java identifiers. It's used here because the original code I copied from was Java and Android Studio converted the code into Kotlin. If I rename this variable to anything other than `object`, I don't need the backticks. 

I renamed the parameter and removed the backticks since we don't really need it: [c9ce875](https://github.com/onramp-io/twitch-android-project-2/pull/20/commits/c9ce875c940e3cdc3efd49eae69975b717168791)

Kotlin doc: https://kotlinlang.org/docs/java-interop.html#escaping-for-java-identifiers-that-are-keywords-in-kotlin

Comment (brownyte) [reply app/src/main/java/com/example/voyagerx/adapters/LaunchCarouselAdapter.kt]: Ooooh, those are backticks. I need to get my glasses prescription updated, haha. Thank you so much for the explanation; that's how you can get around using reserved words - duly noted. Thanks again!!

**Pull request merged**
